tools/perfscale-load-generator/config/config.go

```diff
@@ -0,0 +1,74 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 IBM, Inc.
+ *
+ */
+
+package config
+
+import (
+	"time"
+
+	"k8s.io/client-go/tools/clientcmd"
+
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/tools/perfscale-load-generator/flags"
+)
+
+const RequestTimeout = 15 * time.Second
+
+// NewConfigSpec returns a new config spec object
+func NewConfigSpec() *ConfigSpec {
+	return &ConfigSpec{
+		ClientSet: getClient(),
+		Scenario: Scenario{
+			UUID:                 flags.UUID,
+			Name:                 flags.Name,
+			Namespace:            flags.Namespace,
+			NamespacedIterations: flags.NamespacedIterations,
+			VMISpec: VMISpec{
```

Comment:
This sounds more like it's the ScenarioSpec?
If we change this up a little it would also be able to run different scenarios like creating VMs instead of VMIs to verify performance on different parts of the project.

Reply:
Oh, looks like the GitHub Mobile formatting mislead me here. The scenario specs are below. I see.

Then still I'd prefer to have a more flexible way that lets us create different kind of Kubevirt (or any) resources. VM and VMI for now.